Protacon.RxMq.AzureServiceBus/Topic/AzureTopicPublisher.cs

```diff
@@ -86,14 +86,105 @@ public void Dispose()
                 .ToList()
                 .ForEach(x => x.Dispose());
         }
+
         public Task SendAsync<T>(T message) where T : new()
         {
             var topic = _settings.TopicNameBuilder(message.GetType());
 
             if (!_bindings.ContainsKey(topic))
-                _bindings.Add(topic, new Binding(_settings, _topicManagement, topic, typeof(T), _logger));
-
+            {
+                return TryCreateBinding(topic, typeof(T), message, 5, 60);
+            }
+            
             return _bindings[topic].SendAsync(message);
         }
+
+        /// <summary>
+        /// Creates new Binding for topic in a safe way. If initial tries fail, it will fallback to intervaled retry's.
+        /// </summary>
+        /// <param name="topic"></param>
+        /// <param name="type"></param>
+        /// <param name="message"></param>
+        /// <param name="instantRecoveryTries"></param>
+        /// <param name="lifeCycleRecoveryInterval"></param>
+        private async Task TryCreateBinding(string topic, Type type, object message, int instantRecoveryTries, int lifeCycleRecoveryInterval)
```

Comment:
Is there something special that prevents to use simple retry until success pattern? Retry tries and delays for momen after initial retry.

For example RetryOnException(() => TryBinding(...), TimeSpan interval)

One example of such async implemantation: https://alastaircrabtree.com/implementing-the-retry-pattern-for-async-tasks-in-c/ which can be used as utility method outside.